File: Regular expressions in Python/step02/2.02/2.2.02.py
'''

На вход программе подаётся целое число. Выведите в консоль строку вида: a * a = b, где a - полученное число, а b - результат произведения числа a на само себя.

Sample Input 1:

1

Sample Output 1:

1 * 1 = 1

Sample Input 2:

100

Sample Output 2:

100 * 100 = 10000

Sample Input 3:

5

Sample Output 3:

5 * 5 = 25

Sample Input 4:

36

Sample Output 4:

36 * 36 = 1296

'''

a = int(input())
print(f'{a} * {a} = {a*a}')

File: JavaScript. A3 Tasks/2  Информатикс/2.02/2.2.02.js
/*

Следующее предыдущее

Напишите программу, которая считывает целое число и выводит текст, аналогичный приведенному в примере.

Пробелы, знаки препинания, заглавные и строчные буквы важны!

Входные данные

Вводится целое число.

Выходные данные

Выведите ответ на задачу.

Sample Input 1:

179

Sample Output 1:

The next number for the number 179 is 180.
The previous number for the number 179 is 178.

Sample Input 2:

2

Sample Output 2:

The next number for the number 2 is 3.
The previous number for the number 2 is 1.

*/


let n = Number(prompt());
console.log(`The next number for the number ${n} is ${n+1}.`);
console.log(`The previous number for the number ${n} is ${n-1}.`);
